Resources/public/js/alloyeditor/plugins/floatingtoolbar.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+/* global CKEDITOR */
+YUI.add('ez-alloyeditor-plugin-floatingtoolbar', function (Y) {
+    "use strict";
+
+    if (CKEDITOR.plugins.get('ezfloatingtoolbar')) {
+        return;
+    }
+
+    var FLOATING_TOOLBAR_SELECTOR = '.ae-toolbar-floating',
```

Comment:
The naming convention is a bit different: `SELECTOR_FLOATING_TOOLBAR`

Reply:
I think the convention in Platform is `NAME_SELECTOR`, `NAME_CLASS` so this is correct.

Reply:
We have to start following more appropriate convention. This naming convention is like German. You get the most important information at the end. Until then you don't know what it is.